test/sitemap/test_scrap_sitemap.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+import numpy as np
+import pandas as pd
+import os 
+from quotaclimat.data_ingestion.scrap_sitemap import (find_sections, query_one_sitemap_and_transform)
+from quotaclimat.data_ingestion.config_sitmap import (SITEMAP_CONFIG)
+
+from quotaclimat.data_ingestion.ingest_db.ingest_sitemap_in_db import get_sitemap_list
+
+def test_get_sitemap_list():
+    sitemap = list(get_sitemap_list())[0]
+    # locally we test only a few items
+    sitemap_url = sitemap
+    sitemap_url == "http://nginxtest:80/sitemap_news_figaro_3.xml"
+
+def test_query_one_sitemap_and_transform():
+    sitemap_config = get_sitemap_list()
```

Comment:
Main part of this PR, be able to parse a sitemap.xml from a localhost and be sure about its format.